### PR TITLE
Unify all audio playback commands in one

### LINKED ISSUE #28: Refactor new command to make them async
- [x] Edit commands removing await and changing them to launch

### PR DESCRIPTION
**Pull request by javdc**
One command to rule them all. Use !p to play local sounds, direct urls or other urls supported by yt-dlp. You can even mix different kinds of content in a single line.

Also a new message was added when an unspecified error occurred when running a command.

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -197,24 +199,19 @@ async def search(ctx, arg):
 @bot.command(pass_context=True, aliases=["p"])
 async def play(ctx, *args):
     if len(args) > 999:
-        await ctx.send(":no_entry_sign: No puedes reproducir más de 5 sonidos a la vez.") 
+        await ctx.send(":no_entry_sign: No puedes reproducir tantos sonidos a la vez.")
         return
```
Comment (Wikijito7) [inline main.py:203]: This if statement can be removed.

File: main.py [hunk 2]
```diff
@@ -299,36 +296,20 @@ async def ask(ctx, *args):
...
-
-async def add_url_to_queue(ctx, url, name):
-    if url is None:
-        pass
-    elif is_suitable_for_yt_dlp(url):
-        yt_dlp_info = extract_yt_dlp_info(url)
-        await add_all_found_content_to_queue(ctx, yt_dlp_info)
+    user_voice_channel = get_user_voice_channel(ctx)
+    if user_voice_channel is not None:
+        if len(args) > 0:
```
Comment (Wikijito7) [inline main.py:301]: Add else statement to inform user that at least one argument is required to search.

**Review by Wikijito7 — CHANGES_REQUESTED** [review]

File: main.py (hunk 1 above)
Comment (javdc) [reply main.py:203]: Lo haré en el próximo PR para no mezclar

Comment (javdc) [reply main.py:203]: I'll do it in next PR

**Comment by Wikijito7** [comment]
Waiting for changes in next PR #31

**Pull request merged**